### PR TITLE
N26 component

### PR DESCRIPTION
**Pull request by markusressel**
**Description:**
Documentation for the n26 component.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant/pull/22684

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/n26.markdown [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+sharing: true
+footer: true
+logo: n26.png
+ha_category:
+  - Sensor
+  - Switch
+ha_release: 0.90
+featured: false
+ha_iot_class: "Local Polling"
+redirect_from:
```
Comment (klaasnicolaas) [inline source/_components/n26.markdown]: because the component is new, this is not necessary

File: source/_components/n26.markdown [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+date: 2019-03-11 20:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: n26.png
+ha_category:
+  - Sensor
+  - Switch
+ha_release: 0.90
```
Comment (klaasnicolaas) [inline source/_components/n26.markdown]:
```suggestion
ha_release: "0.90"
```

File: source/_components/n26.markdown [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+## {% linkable_title Configuration %}
+
+Add the following entry to the `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+n26:
+  username: "myn26@email.com"
+  password: "secret"
+  card_id: "0123456789"
```
Comment (klaasnicolaas) [inline source/_components/n26.markdown]:
```suggestion
  card_id: CARD_ID
```

File: source/_components/n26.markdown [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+## {% linkable_title Configuration %}
+
+Add the following entry to the `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+n26:
+  username: "myn26@email.com"
+  password: "secret"
```
Comment (klaasnicolaas) [inline source/_components/n26.markdown]:
```suggestion
  password: YOUR_PASSWORD
```

File: source/_components/n26.markdown [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+The N26 integration for Home Assistant allows you to track your N26 account.
+
+## {% linkable_title Configuration %}
+
+Add the following entry to the `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+n26:
+  username: "myn26@email.com"
```
Comment (klaasnicolaas) [inline source/_components/n26.markdown]:
```suggestion
  username: YOUR_EMAIL
```

File: source/_components/n26.markdown [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+layout: page
+title: "N26"
+description: "Instructions on how to integrate N26 component within Home Assistant."
+date: 2019-03-11 20:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: n26.png
+ha_category:
```
Comment (klaasnicolaas) [inline source/_components/n26.markdown:11]:
```suggestion
ha_category:
  - Finance
```

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_components/n26.markdown (hunk 1 above)
Comment (markusressel) [reply source/_components/n26.markdown]: removed

**Review by klaasnicolaas — DISMISSED** [review]

**Review by klaasnicolaas — DISMISSED** [review]

File: source/_components/n26.markdown [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+  - Finance
+  - Sensor
+  - Switch
+ha_release: "0.91"
```
Comment (klaasnicolaas) [inline source/_components/n26.markdown]:
```suggestion
ha_release: 0.92
```

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_components/n26.markdown (hunk 7 above)
Comment (markusressel) [reply source/_components/n26.markdown]: thx :)

**Review by klaasnicolaas — APPROVED** [review]

**Comment by klaasnicolaas** [comment]
Thnx for the PR! 👍

**Pull request merged**